src/actions/guides/testing/ci-cd.cr

```diff
@@ -0,0 +1,60 @@
+class Guides::Testing::Introduction < GuideAction
+    guide_route "/testing/ci-cd"
+  
+    def self.title
+      "Setting up Continuous Integration"
+    end
+  
+    def markdown : String
+        <<-MD
+        ## Setting up Continuous Integration
+    
+        Continuous integration (CI) is a system that automatically
+        runs your tests every time you check in new code. That way,
+        you can pinpoint the exact change you made that broke the
+        build.
+
+        There are lots of platforms that you can set up to 
+        automatically run tests for you. Below is a list of systems 
+        that are free and popular for open source projects:
+
+        - [Travis CI](https://travis-ci.org/) - The `lucky` CLI 
```

Comment:
Since you mention that Lucky generates this just below, maybe we can leave it off of this spot?